packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -713,7 +758,45 @@ class IosProject extends XcodeBasedProject {
     }
   }
 
-  Future<void> _regenerateFromTemplateIfNeeded() async {
+  Future<void> _updateLLDBIfNeeded() async {
+    if (globals.cache.isOlderThanToolsStamp(lldbInitFile) ||
+        globals.cache.isOlderThanToolsStamp(lldbHelperPythonFile)) {
+      if (isModule) {
+        // When building a module project for Add-to-App, provide instructions
+        // to manually add the LLDB Init File to their native Xcode project.
+        globals.logger.printWarning(
+          'Debugging Flutter on new iOS versions requires an LLDB Init File. '
+          'To ensure debug mode works, please complete one of the following:\n'
+          '  * Set LLDB Init File in your scheme via Xcode > Product > Scheme > '
+          'Edit Scheme for both Run and Test to the following: \n'
+          '      ${lldbInitFile.path}\n'
```

Comment:
Consider adding an empty line before/after content the user needs to copy. It'll make it easier to select the text. I don't feel strongly about this, feel free to skip